ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -143,6 +143,24 @@ tests:
         OPENSHIFT_INSTALL_RELEASE_IMAGE=${RELEASE_IMAGE_LATEST}
         OPENSHIFT_VERSION=4.8
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-kube-api
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        ENABLE_KUBE_API=true
+        NUM_MASTERS=1
```

Comment:
single node? Should be part of the name?

Reply:
currently kube-api is targeted to single-node only
but the single-node is not part of the thing here, we're mainly interested in the kube-api interface here